WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,281 @@
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <li class="nav-item">
+                            <a class="nav-link" href="index.html">The Wallet</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contribution.html">Contribution</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" href="about.html">About us</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="https://blog.wasabiwallet.io" target="_blank">Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="faq.html">Help</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-secondary" href="index.html#download">Download</a>
+                        </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="fixed-img">
+        <img src="img/05_wallet.png" alt="Wasabi Wallet - About">
+        <div class="floating-title">Better money.
+            <br>Better world.</div>
+    </div>
+
+    <div class="content">
+        <section class="blured-section p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8">
+                        <h1 class="title-lined left">Motivation</h1>
+                        <p>
+                            We believe that the most urgent problem for mankind is fixing the money, that Bitcoin is the best option for doing this and that privacy is its biggest problem.
+                        </p>
+                        <p>
+                            Wasabi Wallet was created to help Bitcoin maintain its cypherpunk roots on its path to a global reserve currency and to mitigate the inherent privacy issues of its public ledger. This is why Wasabi Wallet is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop that implements trustless coinjoin magic.
+                        </p>
+                        <p>
+                            After years of doing privacy research and development, we’ve concluded that having good tools is not enough - They need to be easy to use, otherwise they will only be utilized by the minority, making users stand out from the crowd. To increase the use of these tools, Wasabi is designed to be user friendly and to provide privacy automatically by default.
```

Comment:
```suggestion
                            After years of doing privacy research and development, we’ve concluded that having good tools is not enough - They need to be easy to use, otherwise they will only be utilized by the minority, making users stand out from the crowd. To increase the adoption of these tools, Wasabi is designed to be user friendly and to provide privacy automatically by default.
```